client.go

```diff
@@ -50,6 +52,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const PqlVersion = "1.0"
```

Comment:
this should be named PQLVersion according to Go naming conventions

Reply:
Renamed